packages/appsync-emulator-serverless/dynamodbSource.js

```diff
@@ -381,8 +381,7 @@ const batchDeleteItem = async (db, dynamodbTables, { tables }) => {
   };
 };
 
-const resolve = async (dynamodb, defaultAlias, dynamodbTables, payload) => {
-  const table = dynamodbTables[defaultAlias];
```

Comment:
NOTE: This may not be obvious but is a large breaking change. We should encourage use of Ref going forward for aliases of tables. Today the only place where we need/use these alias values is in batch items which *must* always be referred to by name in the queries so we can't use any CF magic there to help.

Reply:
Do we need to worry about our [hack](https://github.com/ConduitVC/aws-utils/blob/66530e7decaf65b1a3a895f536418d0fe61a2a1b/packages/appsync-emulator-serverless/lambdaSource.js#L34) which includes dynamodb table name mappings when running a lambda method?

Reply:
Good question... I will test. I had assumed honestly the test suite would catch it.